frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -73,9 +71,9 @@ action.GitMachete.BaseResetBranchToRemoteAction.info-dialog.title=Git Machete Re
 action.GitMachete.BaseResetBranchToRemoteAction.info-dialog.ok-text=_Reset
 action.GitMachete.BaseResetBranchToRemoteAction.info-dialog.message=\
   <html>\
-    Git Machete Reset performs <b>git reset --keep</b>.<br>\
+    Git Machete Reset performs <b>git reset --keep {0}</b>.<br>\
```

Comment:
As mentioned below, this should be the remote branch, not the local one (actually, changes from https://github.com/VirtusLab/git-machete-intellij-plugin/pull/458 may be needed to fix it)